Firmware/ultralcd.cpp

```diff
@@ -2623,19 +2622,10 @@ static void lcd_menu_xyz_offset()
 {
     lcd_puts_at_P(0, 0, _i("[0;0] point offset"));////MSG_MEASURED_OFFSET c=20
     lcd_puts_at_P(0, 1, separator);
-    lcd_puts_at_P(0, 2, PSTR("X"));
-    lcd_puts_at_P(0, 3, PSTR("Y"));
 
-    float vec_x[2];
-    float vec_y[2];
-    float cntr[2];
-    world2machine_read_valid(vec_x, vec_y, cntr);
-
-    for (uint_least8_t i = 0; i < 2; i++)
-    {
-        lcd_set_cursor((cntr[i] < 0) ? 13 : 14, i+2);
-        lcd_print(cntr[i]);
-        lcd_puts_at_P(18, i + 2, PSTR("mm"));
+    for (uint8_t i = 0; i < 2; i++) {
+        lcd_set_cursor(0, i + 2);
+        lcd_printf_P(PSTR("%c           %6.2fmm"), 'X' + i, eeprom_read_float((float*)(EEPROM_BED_CALIBRATION_CENTER+4*i)));
```

Comment:
:thinking: it looks like the formatting string is correct, there is:
- 1 axis designation character
- 11 spaces
- max 6 digits floating point number (padded with spaces if shorter)
- 2 characters 'mm'

may be we could abuse the float formatting placeholder to avoid the extra spaces as well, something like this:
`"%c%17.2fmm"`

Reply:
Done 👍 Works fine on my end.